__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe.only("3. GET /api/topics", () => {
+  it("status:200, should responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
```

Comment:
Its not really necessary to test that it's an array if you're using a forEach on the array later on, forEach _only_ works on arrays so if line 22 passes you know you have an array. It's good that you considered it though! 

I also want to stress the importance of checking your array has length, maybe use the .toHaveLength matcher. If the output came back as an empty array, it would actually pass in this case, as the iteratee function & expect statement would never be accessed. When using Jest, if you have a describe block with no expect statement, it automatically passes. This means you could get false positives.

Reply:
Thanks, I added toHaveLength matcher underneath it while still keeping the toBeInstanceOf props in there